pages/00-Home.md

```diff
@@ -70,18 +24,31 @@ The hydrus system is built over the following standards and tools:
 
 Other than this, there are also various Python packages that hydrus uses. A list of these packages can be found in the [requirements.txt](https://github.com/HTTP-APIs/hydrus/blob/master/requirements.txt) file.
 
+## Hydra Smart Clients
+
+**Hydra smart client** (`hydra-python-agent`) is a Python implementation of a client based on the Hydra Community Group specifications and reference implementation [Heracles.ts](https://github.com/HydraCG/Heracles.ts). It can navigate a network of hydrus servers autonomously by reading their API documentation. It is a generic client because it can query and retrieve data from any, Hydra-aware or not, server (such as hydrus or any other Web server).
+Every **Hydra smart client** instance provides:
+- A client that can understand Hydra vocabulary and interact autonomously with a Hydra-supporting server via basic [CRUD](https://en.wikipedia.org/wiki/Create,_read,_update_and_delete) operations on data. The contributors are actively developing a fork of `hydra-py` to create a generic [Hydra Agent](https://github.com/HTTP-APIs/hydra-python-agent). 
+
 The Hydra agent is a fork and rebuilding of [hydra-py](https://github.com/pchampin/hydra-py), under development [here](https://github.com/HTTP-APIs/hydra-python-agent).
 
+## Hydra Network
+
+A **Hydra network** is made up of one or more hydrus instances that receives requests from **Hydra smart clients** (or agents, see [`hydra-python-agent`](https://github.com/HTTP-APIs/hydra-python-agent) repository). The interaction of hydrus and Hydra smart clients is how the Hydra ecosystem realizes the Client-Server pattern.
+Generic components of the network are:
+* on hydrus, a REST interface
+* on hydrus, a Hydra-powered server, with the possibilities of being self-deployable and accepting on-the-fly configuration changes
+* on the smart client, a Hydra-powered generic client which can connect to any Hydra-based APIs without hard-coding its behaviour
```

Comment:
hardcoding the api architecture not the client's behavior

Reply:
Yes, I corrected it in the source